samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,307 @@
+const data = [
+    [1451520000000, 0.9363],
+    [1451520000000, 0.99],
```

Comment:
Let's move that data to an HTML file (maybe as CSV? - just thought). If the user would like to check the config of the chart he won't have to scroll through a lot of lines of data.